modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/experience/components/ExperienceModal.js

```diff
@@ -48,22 +48,16 @@ const ExperienceModal = ({
 
 	const [name, setName] = useState(initialName);
 	const [requiredNameError, setRequiredNameError] = useState(false);
-	const [requiredSegmentError, setRequiredSegmentError] = useState(false);
+	const [requiredSegmentError] = useState(false);
```

Comment:
Do we really need a useState here without using the set function? Maybe a simple variable will suffice, or just removing that part of the code

Reply:
Sure, we can get rid of that because that error is not going to happen anymore :) Thanks!